src/test/java/seedu/letsgethired/logic/LogicManagerTest.java

```diff
@@ -128,7 +128,7 @@ private void assertCommandException(String inputCommand, String expectedMessage)
      */
     private void assertCommandFailure(String inputCommand, Class<? extends Throwable> expectedException,
                                       String expectedMessage) {
-        Model expectedModel = new ModelManager(model.getInternTracker(), new UserPrefs());
+        Model expectedModel = new ModelManager(logic.getInternTracker(), new UserPrefs());
```

Comment:
Why was this change introduced as well?

Reply:
logic.getInternTracker() calls model.getInternTracker(), does no change except maintain abstraction barrier.


This is the `logic::getInternTracker`
```
@Override
    public ReadOnlyInternTracker getInternTracker() {
        return model.getInternTracker();
    }
```